composer/loggers/remote_uploader_downloader.py

```diff
@@ -82,17 +106,17 @@ class RemoteUploaderDownloader(LoggerDestination):
             be raised.
 
     Args:
-        object_store_cls (Type[ObjectStore]): The object store class.
+        bucket_uri (str): The remote uri for the bucket to use (e.g. s3://my-bucket).
 
             As individual :class:`.ObjectStore` instances are not necessarily thread safe, each worker will construct
-            its own :class:`.ObjectStore` instance from ``object_store_cls`` and ``object_store_kwargs``.
+            its own :class:`.ObjectStore` instance from ``remote_backend`` and ``backend_kwargs``.
 
-        object_store_kwargs (Dict[str, Any]): The keyword arguments to construct ``object_store_cls``.
+        backend_kwargs (Dict[str, Any]): The keyword arguments to construct the remote backend indicated by ``bucket_uri``.
 
             As individual :class:`.ObjectStore` instances are not necessarily thread safe, each worker will construct
-            its own :class:`.ObjectStore` instance from ``object_store_cls`` and ``object_store_kwargs``.
+            its own :class:`.ObjectStore` instance from ``remote_backend`` and ``backend_kwargs``.
 
-        object_name (str, optional): A format string used to determine the object name.
+        path_format_string (str, optional): A format string used to determine the remote file path (within the specified bucket).
```

Comment:
can we use file somewhere in this name? `file_path`?

Reply:
yeah, fair enough, will use `file_path_format_string`

Reply:
lol I like `path_format_string` -- I think `file` is somewhat implicit. But, I defer to yall :)

Reply:
I'll go with `file_path_format_string`. but as you said Mihir, I guess we should get some team wide style guide around this kind of thing.

Reply:
haha I'm a verbose guy. what can I say?